hudi-common/src/main/java/org/apache/hudi/common/table/HoodieTableMetaClient.java

```diff
@@ -1382,5 +1398,35 @@ public HoodieTableMetaClient initTable(StorageConfiguration<?> configuration, St
         throws IOException {
       return HoodieTableMetaClient.initTableAndGetMetaClient(configuration, basePath, build());
     }
+
+    private void validateMergeConfigs() {
```

Comment:
Where is this method used?

Reply:
I invoke this method after inferring the record merge mode in `HoodieTableMetaClient$PropertyBuilder#build`.